blazor/common/data-binding/webapi-service-binding.md

```diff
@@ -76,11 +252,11 @@ Install-Package Microsoft.EntityFrameworkCore.SqlServer -Version 7.0.11
 {% endtabs %}
 
 ```
-Scaffold-DbContext “Data Source=(localdb)\MSSQLLocalDB;Initial Catalog=OrdersDetails;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False” Microsoft.EntityFrameworkCore.SqlServer -OutputDir Data
+Scaffold-DbContext “Server={Your Server name};Database=OrdersDetails;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False” Microsoft.EntityFrameworkCore.SqlServer -OutputDir Data
```

Comment:
localhost is enough

Reply:
Retained  existing content with "localhost".